sympy/geometry/util.py

```diff
@@ -367,8 +367,8 @@ def convex_hull(*args, polygon=True):
     Optional parameters
     ===================
 
-    polygon : Boolean. If True, returns a Polygon, if false a tuple, see below.
-              Default is True.
+    polygon : bool, default: True
```

Comment:
```suggestion
    polygon : bool, optional
```
I don't think that default value need to be written because it is inspected from argument signature